src/coercions/string.rs

```diff
@@ -25,3 +26,11 @@ impl ToRust<String> for CheckedValue<String> {
         unsafe { std::str::from_utf8_unchecked(slice) }.to_string()
     }
 }
+
+impl ToRuby for String {
+    fn to_ruby(self) -> VALUE {
+        let len = self.len();
+        let cstr = CString::new(self).unwrap();
```

Comment:
This works, but if I understand correctly, this requires two allocations – `CString::new` allocates a C string and memcopy the content there, and then when you pass the pointer to Ruby's `rb_utf8_str_new`, it allocates again and memcpy the content into the Ruby String. Since Ruby already has to do the memcpy anyway, is there any way we can avoid allocating the intermediate C string?